src/components/ForceInclusionCard/index.tsx

```diff
@@ -37,9 +37,9 @@ const ForceInclusionCard: React.FC = () => {
   const l1ChainId = sepolia.id;
 
   const [value, setValue] = useState<string>('');
-  const [data, setData] = useState<`0x${string}`>('0x');
-  const [to, setTo] = useState<Address>('0x');
-  const [gasLimit, setGasLimit] = useState<string>('150000');
+  const [data, setData] = useState<string>('');
+  const [to, setTo] = useState<string>('');
```

Comment:
I changed the type of `to` and `data` to `string` so it does not require a non-empty default value. Checking the value starts with `0x` is done in `errorMessage` and `isInvalid`.